web/styles/settings.css

```diff
@@ -1019,22 +1073,78 @@ input[type="checkbox"] {
         margin: 5px 0 8px;
     }
 
-    .edit_bot_avatar_preview_text {
+    #edit_bot_avatar_preview_text {
         display: none;
     }
 }
 
-#add_bot_preview_text {
-    display: none;
-}
-
 .edit_bot_avatar_preview_image,
 #add_bot_preview_image {
     height: 100px;
     width: 100px;
     margin: 2px 0 8px;
 }
 
+#bot_avatar_save_button {
+    float: right;
+    background-color: hsl(240deg 95% 66%);
+    color: hsl(0deg 0% 100%);
+    margin-top: 9px;
+    transition: transform 0.3s ease;
+}
+
+#bot_avatar_scale_to_fit_button {
+    float: right;
+    background-color: hsl(240deg 95% 66%);
+    color: hsl(0deg 0% 100%);
+    transition: transform 0.3s ease;
+    margin-top: 9px;
+    margin-right: 8px;
```

Comment:
All these repeated 9px/8px margins are very fragile. Can you use CSS grid, or at least CSS variables, for designing this?

Reply:
Yes, we will look into that.